site/content/en/docs/formats/synthia.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'SYNTHIA'
+linkTitle: 'SYNTHIA'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original SYNTHIA dataset is available
+[here](https://synthia-dataset.net).
+
+Supported annotation types:
+- `Mask`
+
+Supported annotation attributes:
+- `dynamic_object` (boolean): whether the object moving
+
+## Import SYNTHIA dataset
+
+A Datumaro project with a SYNTHIA source can be created in the following way:
+
+```bash
+datum create
+datum import --format synthia <path/to/dataset>
+```
+
+It is also possible to import the dataset using Python API:
+
+```python
+from datumaro.components.dataset import Dataset
+
+synthia_dataset = Dataset.import_from('<path/to/dataset>', 'synthia')
+```
+
+SYNTHIA dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+dataset/
+├── GT/
+│   ├── COLOR/
+│   │   ├── Stereo_Left/
+│   │   │   ├── Omni_B
+│   │   │   │   ├── 000000.png
+│   │   │   │   ├── 000001.png
+│   │   │   │   └── ...
+│   │   │   └── ...
+│   │   └── Stereo_Right
+│   │       ├── Omni_B
+│   │       │   ├── 000000.png
+│   │       │   ├── 000001.png
+│   │       │   └── ...
+│   │       └── ...
+│   └── LABELS
+│       ├── Stereo_Left
+│       │   ├── Omni_B
+│       │   │   ├── 000000.png
+│       │   │   ├── 000001.png
+│       │   │   └── ...
+│       │   └── ...
+│       └── Stereo_Right
+│           ├── Omni_B
+│           │   ├── 000000.png
+│           │   ├── 000001.png
+│           │   └── ...
+│           └── ...
+└── RGB
+    ├── Stereo_Left
+    │   ├── Omni_B
+    │   │   ├── 000000.png
+    │   │   ├── 000001.png
+    │   │   └── ...
+    │   └── ...
+    └── Stereo_Right
+        ├── Omni_B
+        │   ├── 000000.png
+        │   ├── 000001.png
+        │   └── ...
+        └── ...
+```
+
+- `RGB` folder containing standard RGB images used for training.
+- `GT/LABELS` folder containing containing PNG files (one per image).
+  Annotations are given in three channels. The red channel contains
+  the class of that pixel. The green channel contains the class only
+  for those objects that are dynamic (cars, pedestrians, etc.),
+  otherwise it contains `0`.
```

Comment:
`0` is also a class. Doesn't this imply that "void" is considered to be dynamic?

Reply:
Apparently, but I think there's no need to put any meaning into it.

Reply:
Well, for the sake of not producing nonsense data, I think it would make sense to update the extractor to not consider 0 a dynamic class.

Reply:
Fixed.